src/GitHub.App/ViewModels/RepositoryPublishViewModel.cs

```diff
@@ -157,18 +150,14 @@ IObservable<ProgressState> OnPublishRepository(object arg)
             var account = SelectedAccount;
 
             return repositoryPublishService.PublishRepository(newRepository, account, SelectedHost.ApiClient)
-                .Select(_ =>
-                {
-                    vsServices.ShowMessage("Repository published successfully.");
-                    return ProgressState.Success;
-                })
+                .Select(_ => ProgressState.Success)
```

Comment:
Oh, yeah, good catch. That message needs to be shown by the ui view, or by the controller that shows the form in the first place, based on the ProgressState result. I missed adding it there I think, need to fixicate.